demo/src/vendor.ts

```diff
@@ -0,0 +1,6 @@
+import 'angular2/bundles/angular2-polyfills';
```

Comment:
Soooo, we _really_ don't want to do this, see https://github.com/angular/angular/issues/5980